apps/block_scout_web/assets/js/lib/csv_download.js

```diff
@@ -28,39 +29,35 @@ const _instance2 = new Pikaday({
 })
 
 $button.on('click', () => {
-  $button.addClass('spinner')
   // eslint-disable-next-line
-  const resp = grecaptcha.enterprise.getResponse()
-  if (resp) {
-    $.ajax({
-      url: './captcha?type=JSON',
-      type: 'POST',
-      headers: {
-        'x-csrf-token': $('[name=_csrf_token]').val()
-      },
-      data: {
-        type: 'JSON',
-        captchaResponse: resp
+  const recaptchaResponse = grecaptcha.enterprise.getResponse()
+  if (recaptchaResponse) {
+    $button.addClass('spinner')
+    $button.prop('disabled', true)
+    $message.removeClass('hidden')
+    const downloadUrl = `${$button.data('link')}&recaptcha_response=${recaptchaResponse}`
```

Comment:
have we tested this on our branch? IIRC there are significant api differences between recaptcha enterprise (what we use) vs upstream recaptcha

Reply:
I tested transactions and epoch transactions downloads on staging.